content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -19,6 +19,8 @@ in that there is no corresponding InfluxDB OSS release.
 - Support user-defined *node labels*.
   Node labels let you assign arbitrary key-value pairs to meta and data nodes in a cluster.
   For instance, an operator might want to label nodes with the availability zone in which they're located.
+- Improve performance of `SHOW SERIES CARDINALITY` and `SHOW SERIES CARDINALITY from <measurement>` InfluxQL queries.
+  These queries now return a `cardinality estimation` column header.
```

Comment:
Previously they returned `count`.

Reply:
I think we have a mix of improve and improved here... these are all past tense since they are done in the release.  Let's make it consistent?